src/tests/ftest/daos_test/rebuild.py

```diff
@@ -284,3 +284,19 @@ def test_rebuild_30(self):
         :avocado: tags=DaosCoreTestRebuild,daos_test,daos_core_test_rebuild,test_rebuild_30
         """
         self.run_subtest()
+
+    def test_rebuild_35(self):
+        """Jira ID: DAOS-16027
+
+        Test Description:
+            Run daos_test -r -s5 -u subtests=35
+
+        Use cases:
+            Core tests for daos_test rebuild
+
+        :avocado: tags=all,daily_regression
+        :avocado: tags=hw,medium
+        :avocado: tags=unittest
```

Comment:
Recommend rebuild tag here, so it's ran with `Features: rebuild`
```suggestion
        :avocado: tags=unittest,rebuild
```